dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -117,7 +117,11 @@ protected void forkRepositoriesFound(Multimap<String, String> pathToDockerfilesI
 
                 // fork the parent if not already forked
                 if (!parentReposForked.contains(parentRepoName)) {
-                    dockerfileGitHubUtil.closeOutdatedPullRequestAndFork(parent);
+                    GHRepository fork = dockerfileGitHubUtil.closeOutdatedPullRequestAndFork(parent);
+                    if (fork == null) {
+                        log.info("Could not fork {}", parentRepoName);
+                        pathToDockerfilesInParentRepo.remove(parentRepoName, c.getPath());
+                    }
                     parentReposForked.add(parentRepoName);
```

Comment:
Should we be adding to `parentReposForked` if this repo was not forked?

Reply:
`parentReposForked` is not used outside of this method. It will avoid trying to create fork and failing multiple times if we are updating multiple Dockerfiles in a same fork.